packages/playground/src/dashboard/node_installer_view.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <div>
+    <iframe :src="ruta" height="1400" width="100%"></iframe>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from "vue";
+
+export default defineComponent({
+  data() {
+    return {
+      ruta: "https://bootstrap.grid.tf/",
```

Comment:
Please rename this to `url`

Reply:
Updated from ruta to url. Please let me know if there's anything else.